### PR TITLE
update cice5

### LINKED ISSUE #33: Restart reproducibility
UFS-S2S must be able to reproduce on restart. The focus should be on achieving this for the CMEPS mediator.

### LINKED ISSUE #44: timestamping of field dumps from caps and mediator
To help diagnose restart reproducibility, it is useful to have 1) the field dumps time-stamped with the relevant import or export time  and 2) all fields for a component in a single file. For example the file "field_ice_export_2012-01-01T00:07:30.nc"  would contain all fields exported by the ice component at time 07:30. 

This will involve changes to all the relevant caps as well as the NEMS mediator. These changes will only affect the case when field dumping is enabled.

### PR DESCRIPTION
**Pull request by DeniseWorthen**
Issue #33:

- When running with CMEPS in S2S, these changes are required for restart reproducibility in ufs-s2s-model.

- An associated change is required in the ice_in file, setting restart_ext to true. 

Related UFS-S2S-model Issue #44:

- When field dumping is turned on, CICE5 will produce a single timestamped files at each import or export time containing all import or export fields. 

Issue #76 

- Adds back missing debug setting

This PR will change all baselines for ufs-s2s-model

Related CICE5 PR [#19](https://github.com/NOAA-EMC/CICE5/pull/19)

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
Baselines on hera and orion were created and tested; all passed except the 6h debug test.  At that point I realized that the 6 hour debug test baseline had failed  at hour 2 with the same error as previously noted in issue #69. 

A fix has been found and a patch to cice is currently awaiting review. Once it is approved, I will re-create the baselines.

**Comment by DeniseWorthen** [comment]
This PR ready for review. New baselines have been created on Hera. Orion baselines will be created and logs added after the system maintenance issue has been resolved.

File: tests/rt.sh [hunk 1]
```diff
@@ -194,6 +194,7 @@ elif [[ $MACHINE_ID = hera.* ]]; then
   ECF_PORT=$(( $(id -u) + 1500 ))
   QUEUE=batch
 #  ACCNR=fv3-cpu
+  ACCNR=marine-cpu
```
Comment (junwang-noaa) [inline tests/rt.sh]: Do you want to change it back to fv3_cpu?

**Review by junwang-noaa — APPROVED** [review]

File: tests/rt.sh (hunk 1 above)
Comment (JessicaMeixner-NOAA) [reply tests/rt.sh]: Isn't any non-GFSv16 work not supposed to be using fv3_cpu?

Comment (MinsukJi-NOAA) [reply tests/rt.sh]: Actually, fv3-cpu is commented out here originally (not by Denise). Default is set in detect_machine.sh as nems.

Comment (junwang-noaa) [reply tests/rt.sh]: sorry, yes, that is what I say, we do not need to set ACCNR,

Comment (DeniseWorthen) [reply tests/rt.sh]: I think I have the same issue on hera as on orion, which is that I didn't have project nems available so detect machine does not work. I will remove the accnr=marine-cpu.

**Review by JessicaMeixner-NOAA — APPROVED** [review]

**Review by binli2337 — APPROVED** [review]

**Pull request merged**